app/_data/docs_nav_kuma_2.2.x.yml

```diff
@@ -553,6 +553,17 @@ items:
             url: "/policies/meshloadbalancingstrategy/#configuration"
           - text: Examples
             url: "/policies/meshloadbalancingstrategy/#examples"
+  - title: Guides
```

Comment:
Are you sure everything here is applicable to `2.2.x`? also `2.5.x` goes out of support on `2024-11-15` so should we really be modifyting `2.2.x - 2.5.x`? WDYT?

Reply:
addressed